src/bootstrap-table.js

```diff
@@ -283,6 +283,7 @@ class BootstrapTable {
           Utils.sprintf(' rowspan="%s"', column.rowspan),
           Utils.sprintf(' colspan="%s"', column.colspan),
           Utils.sprintf(' data-field="%s"', column.field),
+          Utils.sprintf(' id="%s"', column.field),
```

Comment:
Is the `id` attribute required? If there are multiple tables at the same time, the`id` may be the same.
> [The id global attribute defines an identifier (ID) which must be unique in the whole document.](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id)

Reply:
Yes the id is required for the sorting, but your right.
Should we move the[cookieidtable](https://bootstrap-table.com/docs/extensions/cookie/#cookieidtable) to the core (without cookie prefix), so we can always determinate the table.
In that case i would use the idtable-field